package.json

```diff
@@ -7,12 +7,12 @@
     "bulma": "^0.5.3",
     "draft-js": "^0.10.4",
     "jwt-decode": "^2.2.0",
-    "draft-js": "^0.10.4",
     "prop-types": "^15.6.0",
     "react": "^16.0.0",
     "react-dom": "^16.0.0",
     "react-dropzone": "^4.1.3",
     "react-redux": "^5.0.6",
+    "react-router": "^4.2.0",
```

Comment:
I don't think we need full react-router, react-router-dom should suffice. I'll investigate now before approving.